src/Manager.js

```diff
@@ -9,6 +9,9 @@ class Manager {
     this.id = managerData.id
```

Comment:
What will be the manager id? From what I've seen there isn't an id associated with the manager.